discovery-provider/src/queries/query_helpers.py

```diff
@@ -328,6 +329,81 @@ def populate_track_metadata(session, track_ids, tracks, current_user_id):
     )
     save_count_dict = {track_id: save_count for (track_id, save_count) in save_counts}
 
+
+    remix_query = session.query(
+        Track.owner_id.label('track_owner_id'),
+        Remix.parent_track_id.label('parent_track_id'),
+        Remix.child_track_id.label('child_track_id'),
+        Save.is_current.label('has_remix_author_favorited'),
+        Repost.is_current.label('has_remix_author_reposted'),
+        User
+    ).join(
+        Remix,
+        and_(
+            Remix.parent_track_id == Track.track_id,
+            Remix.child_track_id.in_(track_ids)
+        )
+    ).join(
+        User,
+        and_(
+            User.user_id == Track.owner_id,
+            User.is_current == True
+        )
+    ).outerjoin(
+        Save,
+        and_(
+            Save.save_item_id == Remix.child_track_id,
+            Save.save_type == SaveType.track,
+            Save.is_current == True,
+            Save.is_delete == False,
+            Save.user_id == Track.owner_id
+        )
+    ).outerjoin(
+        Repost,
+        and_(
+            Repost.repost_item_id == Remix.child_track_id,
+            Repost.user_id == Track.owner_id,
+            Repost.repost_type == RepostType.track,
+            Repost.is_current == True,
+            Repost.is_delete == False
+        )
+    ).filter(
+        Track.is_current == True
+    )
```

Comment:
doesn't this query need a .all or something?

Reply:
weird that it doesn't but added